src/common/exporter/scrivener/v2/exporters/beats.js

```diff
@@ -28,15 +28,23 @@ export default function exportBeats(state, documentContents, options) {
   const linesById = keyBy(lines, 'id')
   const customAttrs = cardsCustomAttributesSelector(state)
   const outlineFilter = state.ui.outlineFilter
+  const outlineExportFilter = options.outline.filter
```

Comment:
Isn't this the same as the `outlineFilter`?

Reply:
@Quiescent these are the lines that are checked inside the `Advanced export options` modal when you click the export button then choose `Advanced..` 

They will differ once the FilterCheckOption is changed, it will push the `lineId` to the `options.outline.filter`. 

If there's an `outlinefilter` those `lineId` will be selected by default, and if the user select more plotlines, all those ids (including the `outlineFilter`) will be stored to `options.outline.filter`.

`options.outline.filter` is empty if the user will not click the `Advanced` export option and will only export the the `lineId` from the `outlineFilter` from the store, if no outline filter then the entire outline will be exported.

Reply:
Cool.  That sounds sensible.

One question: does the outline filter (when you're done exporting and head back to the outline) change according to the options that you selected in the exporter?

Reply:
The `outlineFilter` from store will be the same, but the `options.outline.filter` from the `Advanced export` modal will be set to null since the number of plotlines can vary but other settings are saved by default (same on prod).

Reply:
I'm not sure that I understand.  Are you saying that the `outlineFilter` in the store is set to the same value as what you choose in the `Advanced Export`, but that the `options.outline.filter` is set to `null` every time that we export?
What happens to `options.outline.filter` if the user turns on the "Save these settings across projects" setting?

Reply:
> I'm not sure that I understand. Are you saying that the outlineFilter in the store is set to the same value as what you choose in the Advanced Export, but that the options.outline.filter is set to null every time that we export?

Yes, the `outlineFilter` in the store (if there is) will be the selected values of the `options.outline.filter`, if `outlineFilter` is null, then we select all the `lineIds` to be exported.

If the user select/check another `plotline` to be exported, the `outlineFilter` values will be mapped to the `options.outline.filter` plus the new `plotline` that was selected. 

line 34 will be true
```
if (outlineExportFilter) {      return cards.filter((card) => includes(outlineExportFilter, card.lineId))    } 
```

> but that the options.outline.filter is set to null every time that we export?
> What happens to options.outline.filter if the user turns on the "Save these settings across projects" setting?

Yes `options.outline.filter` will always be null every after export and they will not be saved even if the user turns on the "Save these settings across projects" setting
Because each story has different number of plotlines / `lineId`, the export will be broken if we will save the line especially when the previous pltr file has more plotlines